pkg/authenticator/username_test.go

```diff
@@ -0,0 +1,24 @@
+package authenticator
+
+import (
+	"testing"
+
+	. "github.com/smartystreets/goconvey/convey"
+)
+
+func TestUsername(t *testing.T) {
```

Comment:
Is there a standard for tests names? Like UsernameSplittedSuccessfully ?
What can I know if I see TestUsername failed in the output of the tests?

Reply:
Here is when the Convey clauses below come to action. For example, you will see a failure in:
`NewUsername -> Given a host's username -> Splits the username as expected` and will understand the context and the unexpected output.